packages/react-devtools-shell/webpack.config.js

```diff
@@ -107,7 +107,7 @@ const config = {
             options: {
               sourceMap: true,
               modules: true,
-              localIdentName: '[local]___[hash:base64:5]',
+              localIdentName: '[local]',
```

Comment:
Normally I think use something more stable and intentional, like test IDs.

But since this change only impacts the testing shell, it seems okay for now. :)